test262_errors.txt

```diff
@@ -79,6 +79,7 @@ test262/test/language/expressions/in/private-field-invalid-assignment-target.js:
 test262/test/language/expressions/in/private-field-invalid-assignment-target.js:23: strict mode: unexpected error type: Test262: This statement should not be evaluated.
 test262/test/language/expressions/object/computed-property-name-topropertykey-before-value-evaluation.js:31: Test262Error: Expected SameValue(«"bad"», «"ok"») to be true
 test262/test/language/expressions/object/computed-property-name-topropertykey-before-value-evaluation.js:31: strict mode: Test262Error: Expected SameValue(«"bad"», «"ok"») to be true
+test262/test/language/module-code/export-expname-string-binding.js:19: unexpected error type: Test262: This statement should not be evaluated.
```

Comment:
This one should SyntaxError because `export { "foo" as "bar" }` is not valid syntax (but `export { "foo" as "bar" } from "path/to/wherever"` is), right? Should be easy to fix, I think?